src/Components/Header.js

```diff
@@ -2,29 +2,29 @@ import React from 'react';
 
 import HeptagramLogo from '../Assets/full-heptagram-logo.png';
 import Hamburger from './Hamburger';
-
+import {Link ,Navigate,useNavigate} from 'react-router-dom';
```

Comment:
Hey, please remove Navigate & useNavigate here, as they are unused variables.

```suggestion
import {Link} from 'react-router-dom';
```

Reply:
Ok , @j-dogcoder